src/rpc/util.h

```diff
@@ -22,6 +22,8 @@
 
 #include <boost/variant.hpp>
 
+static const std::string UNIX_EPOCH_TIME = "UNIX epoch time";
```

Comment:
needs a comment
```
/** String used to describe UNIX epoch times in documentation, factored out to a constant for consistency. */
```
or something

Reply:
I also think defining the value of a string constant in the header can give linker issues in C++, or at least increases executable size because this will be in every compilation unit.

usually you'd do in `util.h`:
```c++
extern const std::string UNIX_EPOCH_TIME;
```
the in `util.cpp`
```
const std::string UNIX_EPOCH_TIME = "UNIX epoch time";
```

Reply:
Thanks! Done, LMK if it would be better to comment the header rather than the definition.